egs/librispeech/asr/simple_v1/README.md

```diff
@@ -0,0 +1,132 @@
+
+
+## k2_librispeech
+An example of how to build G and L FST for K2.
+
+Most scripts of this example are copied from Kaldi.
+
+### Run scripts
+```bash
+$ ./run.sh
+
+$ ls data/lang_nosp
+
+G.fsa.txt
+L.fst.txt
+L_disambig.fst.txt
+oov.int
+oov.txt
+phones
+phones.txt
+words.txt
+```
+
+### Load L, G into K2
+```python
+import k2, _k2
```

Comment:
curious - why do you need to import (seemingly private) `_k2`?

Reply:
I would suggest removing `_k2`. Users are not supposed to use it directly. 

`_k2` is largely undocumented.

Reply:
Deleted most of content of this file (they were copied from @jimbozhang 's L and G preparing scripts, and were just examples junbo wrote to show how to use them for me)